release/files_common

```diff
@@ -100,6 +100,7 @@ selfdrive/boardd/pigeon.h
 selfdrive/boardd/set_time.py
 
 selfdrive/car/__init__.py
+selfdrive/car/docs.py
```

Comment:
doesn't need to be here

Reply:
We need to import from here in values if we don't want to re-define stuff like Column and CarFootnote